rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -82,6 +85,50 @@ void SqliteStorage::open(
     initialize();
   }
 
+  if (extra_file_) {
+    const bool ro = is_read_only(io_flag);
+
+    // in write mode we add at least one serialized file
+    // (it would be possible to split the data into into multiple files)
+    if(!ro)
+    {
+      data_files_.push_back(
+        std::make_unique<BinarySequentialFile>(database_name_ + ".extra")
+      );
+    }
+    else if(metadata->relative_file_paths.size()>=1)
```

Comment:
spaces around the operators

Reply:
how does such a metafile now look like? What's the enforced format for the relative file paths ?